RPIPs/RPIP-36.md

```diff
@@ -0,0 +1,94 @@
+---
+rpip: 36
+title: Committee Membership Record
+description: This RPIP records active committees within the Rocket Pool DAO, alongside their current and historic membership.
+author: LongForWisdom (@LongForWisdom)
+status: Living
+type: Informational
+created: 2023-11-03
+requires: 10
+---
+
+## Abstract
+
+This informational RPIP lists the active committees in the Rocket Pool DAO, gives a brief overview their purpose, and provides links to further information. It also contains current and historic membership information for each of these committees.
+
+
+## Motivation
+
+Currently, there is no overview of the active committees within Rocket Pool, or their purpose. Currently, users need to both know to look and know how to find the relevant RPIPs to find information that can be presented in a more user-friendly way. Further, historical membership is not being recorded in a structued way outside of the committees themselves.  
+
+## Committee Summary
+
+| Committee                             | Chief Mission                                                          | Date Established | Defining RPIPs                         | Multisig Addresses                                                                                                                                                                         | Public Documents                                                                                                                                                                                                                                                                         |
+|---------------------------------------|------------------------------------------------------------------------|------------------|----------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Incentives Management Committee (IMC) | To support low slippage exchange between ETH and rETH                  | 2022-08-25       | [RPIP-20](RPIP-20), [RPIP-10](RPIP-10) | [Mainnet](https://app.safe.global/home?safe=eth:0xb867EA3bBC909954d737019FEf5AB25dFDb38CB9), [Arbitrum](https://app.safe.global/home?safe=arb1:0xd7102A3744c302f167c53621453516345bC460d7) | [Tracking Sheet](https://docs.google.com/spreadsheets/d/1jLGkoiSjNr1UgG1spAH9ppBfrk_FqjJXg9CIw35nLCA/edit#gid=1325210165)                                                                                                                                                                |
```

Comment:
Could consider https://discord.com/channels/405159462932971535/929890788551323678/1163935228205731920 for documents as "Partner Proposal Cheat Sheet" or smth

Reply:
Decided not, will add if it moves out of discord.